ssqatest/tests/backend/misc_tests/test_create_new_product.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+from ssqatest.src.api_helpers.ProductsAPIHelper import ProductsAPIHelper
+from ssqatest.src.dao.products_dao import ProductsDAO
+from ssqatest.src.utilities.genericUtilities import generate_random_string
+import logging as log
```

Comment:
This works but you will see 99% of the time loggers are called 'logger' so caling it 'log' sounds fancy but not common so I woun't do that. Just my opinion.